examples/tv-casting-app/android/README.md

```diff
@@ -21,9 +21,10 @@ the TV.
 ## Requirements for building
 
 You need Android SDK 21 & NDK 23.2.8568313 downloaded to your machine. Set the
```

Comment:
Can you make a change to omit stating any specific version numbers for the Android SDK or NDK, and instead provide a link to this doc: https://github.com/project-chip/connectedhomeip/blob/7a815f675c15e7b9d35bddde1324306ce7887b9e/docs/guides/android_building.md#requirements-for-building